contexts/CharacterActionsContext.tsx

```diff
@@ -123,6 +124,12 @@ export const CharacterActionsProvider: React.FC<React.PropsWithChildren> = ({
     null,
   );
 
+  useEffect(() => {
```

Comment:
Is this wise? There are game master actions which use the selectedCharacter and it may be incorrect if they don't reset it correctly.

Reply:
removed in [7f173e2](https://github.com/raid-guild/character-sheets-frontend/pull/160/commits/7f173e25a640407b6f7e7b1a7febc2d4907f50d5)